server/chat-plugins/info.js

```diff
@@ -2479,37 +2486,35 @@ const commands = {
 
 /** @type {PageTable} */
 const pages = {
-	punishments(query, user, connection) {
+	punishments(query, user) {
 		this.title = 'Punishments';
 		let buf = "";
 		this.extractRoom();
 		if (!user.named) return Rooms.RETRY_AFTER_LOGIN;
 		if (!this.room.chatRoomData) return;
 		if (!this.can('mute', null, this.room)) return;
-		const sortedPunishments = Punishments.getPunishmentsOfRoom(this.room).sort((a, b) =>
-			// Ascending order
-			a.expiresIn - b.expiresIn
-		);
+		// Ascending order
+		const sortedPunishments = Punishments.getPunishmentsOfRoom(this.room).sort((a, b) => a.expiresIn - b.expiresIn);
 		if (sortedPunishments.length) {
-			buf += `<div class="pad"><h2>List of active punishments:</h2>`;
-			buf += `<table style="border: 1px solid black; border-collapse:collapse; width:100%;">`;
+			buf += `<div class="ladder pad"><h2>List of active punishments:</h2>`;
+			buf += `<table">`;
 			buf += `<tr>`;
-			buf += `<th style="border: 1px solid black;">Username</th>`;
-			buf += `<th style="border: 1px solid black;">Punishment type</th>`;
-			buf += `<th style="border: 1px solid black;">Expire time</th>`;
-			buf += `<th style="border: 1px solid black;">Reason</th>`;
-			buf += `<th style="border: 1px solid black;">Alts</th>`;
-			if (user.can('ban')) buf += `<th style="border: 1px solid black;">IPs</th>`;
+			buf += `<th>Username</th>`;
+			buf += `<th>Punishment type</th>`;
+			buf += `<th>Expire time</th>`;
+			buf += `<th>Reason</th>`;
+			buf += `<th>Alts</th>`;
+			if (user.can('ban')) buf += `<th>IPs</th>`;
 			buf += `</tr>`;
 			for (const punishment of sortedPunishments) {
-				let expireString = Chat.toDurationString(punishment.expiresIn, {precision: 1});
+				const expireString = Chat.toDurationString(punishment.expiresIn, {precision: 1});
 				buf += `<tr>`;
-				buf += `<td style="border: 1px solid black;">${punishment.id}</td>`;
-				buf += `<td style="border: 1px solid black;">${punishment.punishType.toLowerCase()}</td>`;
-				buf += `<td style="border: 1px solid black;">${expireString}</td>`;
-				buf += (punishment.reason) ? `<td style="border: 1px solid black;">${punishment.reason}</td>` : `<td style="border: 1px solid black;"> - </td>`;
-				buf += (punishment.alts.length) ? `<td style="border: 1px solid black;">${punishment.alts.join(", ")}</td>` : `<td style="border: 1px solid black;"> - </td>`;
-				buf += (user.can('ban') && punishment.ips.length) ? `<td style="border: 1px solid black;">${punishment.ips.join(", ")}</td>` : (user.can('ban') && !punishment.ips.length) ? `<td style="border: 1px solid black;"> - </td>` : ``;
+				buf += `<td>${punishment.id}</td>`;
+				buf += `<td>${punishment.punishType.toLowerCase()}</td>`;
+				buf += `<td>${expireString}</td>`;
+				buf += `<td>${punishment.reason || ' - '}</td>`;
+				buf += (punishment.alts.length) ? `<td>${punishment.alts.join(", ")}</td>` : `<td> - </td>`;
```

Comment:
I was hoping you would do these like `punishment.reason`

```suggestion
				buf += `<td>${punishment.alts.join(", ") || ' - '}</td>`;
```